tower-http/src/lib.rs

```diff
@@ -37,3 +37,51 @@
 #![forbid(unsafe_code)]
 #![cfg_attr(docsrs, feature(doc_cfg))]
 #![cfg_attr(test, allow(clippy::float_cmp))]
+
+use http::Response;
+
+#[cfg(feature = "trace")]
+#[cfg_attr(docsrs, doc(cfg(feature = "trace")))]
+pub mod trace;
+
+#[derive(Debug, Clone, Copy)]
+pub enum LatencyUnit {
+    Millis,
+    Nanos,
+}
+
+/// You might not always want to log the literal HTTP status. gRPC for example has its own status
+/// and always uses `200 OK` even for errors.
+// TODO(david): can we come up with a better name for this?
+pub trait GetTraceStatus<T, E> {
```

Comment:
nit/TIOLI: IMO, idiomatic rust tends not to use `get` except for collections that can look up arbitrary keys. I might drop the `Get` (but see my other comment about maybe replacing this with a more generic response-classifier design?)

Reply:
Good point. I'll keep this in mind when I think about [your other comment](https://github.com/tower-rs/tower-http/pull/40#discussion_r566474374).